core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerClientProvider.scala

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.dsde.rawls.billing
+
+import bio.terra.profile.api.{AzureApi, ProfileApi}
+import bio.terra.profile.client.ApiClient
+
+/**
+ * Implementors of this trait know how to instantiate billing profile manager client
+ * classes for use with the billing profile manager service.
+ */
+trait BillingProfileManagerClientProvider {
+  def getApiClient(accessToken: String): ApiClient
+
+  def getAzureApi(accessToken: String): AzureApi
+
+  def getProfileApi(accessToken: String): ProfileApi
```

Comment:
I would suggest calling this `getBillingProfileApi`. When I first read this trait, I was confused as to what profile referred to (given that we also have other types of profiles in Terra).

Reply:
The type as exposed by BPM is the `ProfileApi`, we should keep those in sync.